packages/model-viewer/src/features/scene-graph/sampler.ts

```diff
@@ -146,15 +137,15 @@ export class Sampler extends ThreeDOMElement implements SamplerInterface {
   }
 
   get rotation(): number {
-    return this[$sourceSampler].rotation;
+    return [...this[$threeTextures]][0]['rotation'];
   }
 
   get scale(): Vector2 {
-    return this[$sourceSampler].repeat;
+    return [...this[$threeTextures]][0]['repeat'];
```

Comment:
Also, prefer e.g. `.repeat` to `['repeat']` whenever possible. The latter defeats TS typing.